app/classifier/feedback/process-feedback-drawing.js

```diff
@@ -11,6 +11,7 @@ const isWithinTolerance = (annotationX, annotationY, feedbackX, feedbackY, toler
 
 const createFeedbackItem = (success, annotation, rule) => {
   const feedbackItem = {
+    frame: annotation.value.map(item => parseInt(item.frame)),
```

Comment:
Again, `Array.map` returns an array, so reduce it down to a single value to get a frame number.